SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -1009,6 +1010,7 @@ - (void)sendMessageToService:(TSOutgoingMessage *)message
             failure:^(NSURLSessionDataTask *task, NSError *error) {
                 NSHTTPURLResponse *response = (NSHTTPURLResponse *)task.response;
                 NSInteger statusCode = response.statusCode;
+                NSData *_Nullable responseData = error.userInfo[AFNetworkingOperationFailingURLResponseDataErrorKey];
```

Comment:
AFNetworking exposes error response bodies using `AFNetworkingOperationFailingURLResponseDataErrorKey`, but we need to include them in the failure callback.